backport.py

```diff
@@ -269,6 +279,12 @@ def main(args):
     else:
         assert False
 
+    if args.pr == args.sha and args.pr is None:
```

Comment:
Do you mean the following?
```suggestion
    if args.pr is None and args.sha is None:
```

Reply:
thought it was ok but certainly, this is better. Thanks again